go-app-ussd_popi_faq.js

```diff
@@ -63,32 +61,7 @@ go.app = function() {
             ;
         };
 
-        self.add = function(name, creator) {
-            self.states.add(name, function(name, opts) {
-                if (!interrupt || !utils.timed_out(self.im))
-                    return creator(name, opts);
-
-                interrupt = false;
-                var timeout_opts = opts || {};
-                timeout_opts.name = name;
-                return self.states.create('state_timed_out', timeout_opts);
-            });
-        };
-
-        self.states.add('state_timed_out', function(name, creator_opts) {
-            return new ChoiceState(name, {
-                question: $('Welcome back. Please select an option:'),
-                choices: [
-                    new Choice(creator_opts.name, $('Continue viewing FAQ')),
-                    new Choice('state_start', $('Main menu'))
-                ],
-                next: function(choice) {
-                    return choice.value;
-                }
-            });
-        });
```

Comment:
@rudigiesler I'm confused as to why the reduced space in the first message would require us to have the timeout states? because they'd have more screens to traverse?